tests/units/Model/ColorModelTest.php

```diff
@@ -11,8 +11,6 @@ public function testFind()
     {
         $colorModel = new ColorModel($this->container);
         $this->assertEquals('yellow', $colorModel->find('yellow'));
-        $this->assertEquals('yellow', $colorModel->find('Yellow'));
-        $this->assertEquals('dark_grey', $colorModel->find('Dark Grey'));
```

Comment:
Why do you remove these assertions?

Reply:
These assertions are a duplicate of code.

Reply:
This is not a duplicate. Pay attention to the character case.